### PR TITLE
Expose default vscode extensions

### PR DESCRIPTION
**Pull request by CGNonofr**
based on https://github.com/CodinGame/monaco-vscode-api/pull/94

the package now exposes some module that register vscode default extensions:

for instance:
```typescript
import 'vscode/default-extensions/theme-defaults'
import 'vscode/default-extensions/javascript'
```

is now enough to have the default vscode themes and the javascript language

### CODE REVIEW

**Comment by kaisalmen** [comment]
This sounds good. I will review it tomorrow.

**Comment by CGNonofr** [comment]
@kaisalmen I've also added a rollup/vite plugin that allows to load a vsix file directly (only the manifest or course, not the code)

**Comment by kaisalmen** [comment]
@CGNonofr I thought this is independent, but now I looked at the git graph and I have to start with
https://github.com/CodinGame/monaco-vscode-api/pull/93 right?

**Comment by CGNonofr** [comment]
> @CGNonofr I thought this is independent, but now I looked at the git graph and I have to start with #93 right?

Yep, every PR is based on the previous one

Alsmost everytime I make changes I need to tweak the polyfills so I prefer to be based on the last working version to not have to rebase all remaining PRs and test them when one is merged

**Comment by CGNonofr** [comment]
I've published a [1.78.0-next.0](https://www.npmjs.com/package/@codingame/monaco-vscode-api/v/1.78.0-next.0) including all the open PRs

**Comment by kaisalmen** [comment]
@CGNonofr Doesn't it make sense to only look at the delta between main and #98 because otherwise I check things that are gone a couple of commits later. WDYT?

**Comment by CGNonofr** [comment]
> eworked a couple of comm

The PRs are based on each other but quite still independent. I don't think there is anything done then undone, the commits are mostly well splitted. I'm not sure you'll be able to understand everything by just looking at the whole changes.

But It's a matter of preferences, do as you wish

File: README.md [hunk 1]
```diff
@@ -135,15 +140,62 @@ npm install -D @types/vscode
...
+```
+
+### Loading vsix file
+
+VSCode extension are bundled as vsix files.
+This library exposes a rollup plugin (vite-compatible) that allows to load a vsix file. The code is not used, only the declarative part in the manifest.
+
+- rollup/vite config:
+```typescript
+import vsixPlugin from 'vscode/rollup-vsix-plugin'
```
Comment (kaisalmen) [inline README.md:187]: Have you already made use of it somewhere?

**Review by kaisalmen — APPROVED** [review]
I looked at the real delta locally and let's go. 🎉

**Comment by kaisalmen** [comment]
> The PRs are based on each other but quite still independent. I don't think there is anything done then undone, the commits are mostly well splitted. I'm not sure you'll be able to understand everything by just looking at the whole changes.

Yes, looking at them separately, but with an external tool to see the real delta in between the branches, was the better approach.

File: README.md (hunk 1 above)
Comment (CGNonofr) [reply README.md:187]: I did on the demo with the ocaml vscode extension. The demo already contains a lot of stuff and I don't really know how to showcase it properly

**Pull request merged**